logic/build.gradle.kts

```diff
@@ -93,9 +93,19 @@ kotlin {
                 implementation(libs.work)
             }
         }
+        val androidUnitTest by getting {
```

Comment:
I'm having problems to see this new test execution on the CI reports, can you confirm, or maybe I'm just searching in the wrong place 😅 

I think on CI, we just run `jvmTest` and `jsTest` (tried locally and don't see it either 😓) we might need to add this target exec to our pipeline as well :D